frontend/packages/git-service/src/services/bitbucket-service.ts

```diff
@@ -108,7 +122,9 @@ export class BitbucketService extends BaseService {
   };
 
   isFilePresent = async (path: string): Promise<boolean> => {
-    const url = `${this.baseURL}/repositories/${this.metadata.owner}/${this.metadata.repoName}/src/${this.metadata.defaultBranch}/${path}`;
+    const url = this.isServer
+      ? `${this.baseURL}/projects/${this.metadata.owner}/repos/${this.metadata.repoName}/browse/${path}?at=${this.metadata.defaultBranch}`
+      : `${this.baseURL}/repositories/${this.metadata.owner}/${this.metadata.repoName}/src/${this.metadata.defaultBranch}/${path}`;
```

Comment:
Got on 401 when trying `https://bitbucket.localhost:22222/projects/ODC/repos/nodejs-ex` (with our Caddyfile) because there are two slashes before package.json:

```suggestion
    const url = this.isServer
      ? `${this.baseURL}/projects/${this.metadata.owner}/repos/${this.metadata.repoName}/browse/${path}?at=${this.metadata.defaultBranch}`
      : `${this.baseURL}/repositories/${this.metadata.owner}/${this.metadata.repoName}/src/${this.metadata.defaultBranch}${path}`;
```